src/redismodule.h

```diff
@@ -873,6 +873,7 @@ typedef struct RedisModuleTypeMethods {
 REDISMODULE_API void * (*RedisModule_Alloc)(size_t bytes) REDISMODULE_ATTR;
 REDISMODULE_API void * (*RedisModule_TryAlloc)(size_t bytes) REDISMODULE_ATTR;
 REDISMODULE_API void * (*RedisModule_Realloc)(void *ptr, size_t bytes) REDISMODULE_ATTR;
+REDISMODULE_API size_t (*RedisModule_AllocUsableSize)(void *ptr) REDISMODULE_ATTR;
```

Comment:
you're missing a mention of this new API both at the bottom of redismodule.h and at the bottom of module.c.
they're both without an `RM_` prefix (look for MallocSize in these files)

Reply:
done